project/app/.eslintrc.yaml

```diff
@@ -1,4 +1,4 @@
 ---
 extends: availity/browser
 rules:
-  unicorn/prefer-query-selector: off
\ No newline at end of file
+  unicorn/prefer-query-selector: off
```

Comment:
i don't believe there is a linter issue with this as the `index.js` uses `queryselector`.

Reply:
Not sure what is going on here. Files look the same:
https://github.com/djejaquino/availity-starter-react/blob/master/project/app/.eslintrc.yaml
https://github.com/Availity/availity-starter-react/blob/master/project/app/.eslintrc.yaml

Reply:
They are closely similar but if there is ever an event where you have things in the `data`, `config` or `resource` folders that you want the linter to ignore you can put the rules at the root and it will only apply to those. After it reaches the app code the nested `eslint.yaml` will take over.

I personally haven't seen a use case for this yet and we can possible just remove the nested one as a whole and stick with the single `eslintrc.yaml` at the root.

Reply:
i have personally seen it useful for plop files etc etc at root of directory

Reply:
@GoPro16  I meant to restore original file since you guys found a better solution